ts/main.ts

```diff
@@ -15,25 +15,49 @@ const animeContainer = document.querySelector(
   '.anime-container',
 ) as HTMLDivElement;
 const detailView = document.querySelector('.detail-view') as HTMLDivElement;
+const favoritesLink = document.querySelector(
+  '.favorite-page-title',
+) as HTMLAnchorElement;
+
+let favorites: Anime[] = JSON.parse(localStorage.getItem('favorites') || '[]');
```

Comment:
Ideally you would have a `readData`/`writeData` type of functions like we did in `typescript-local-storage` to make it more readable/reusable and then this would be in your `data.ts` file.